src/odemis/util/comp.py

```diff
@@ -15,8 +15,10 @@
 You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
 '''
```

Comment:
could you update this to triple double quotes, while your cleaning up the document anyway?